lib/generate-query.d.ts

```diff
@@ -20,6 +21,8 @@ export declare function generateRandomMutation(schema: GraphQLSchema, config?: C
         [varName: string]: any;
     };
     seed: number;
+    typeCount: number;
```

Comment:
I do not recall the purpose of `typeCount` and `resolveCount` and why it is important to expose it. In any case, this documentation is missing.

Reply:
I changed it mostly for consistency's sake. There is actually quite a bit of code duplication that could be tackled at some point. I also think we might rename `resolveCount` to `fieldCount`, as this is about queries. The purpose is to get a sense of the dimensions of the generated queries / mutations.

Reply:
Ah I see, that makes a lot more sense now.